tidb-configuration-file.md

```diff
@@ -438,6 +438,51 @@ The Plan Cache configuration of the `PREPARE` statement.
 - The threshold of the TiKV load. If the TiKV load exceeds this threshold, more `batch` packets are collected to relieve the pressure of TiKV. It is valid only when the value of `tikv-client.max-batch-size` is greater than `0`. It is recommended not to modify this value.
 - Default value: `200`
 
+### `enable-one-pc` <!-- New in v5.0 -->
+
+- Specifies whether to use the first phase commit feature for transactions involving only one Region. Compared with the traditional two-phase commit, first phase commit can greatly reduce transaction commit latency and increase throughput.
```

Comment:
It should be "one-phase commit", in contrast to the traditional "two-phase commit".